Quaver.Shared/Database/Maps/OtherGameMapDatabaseCache.cs

```diff
@@ -176,13 +176,25 @@ private static List<OtherGameMap> SyncMaps()
             if (string.IsNullOrEmpty(ConfigManager.EtternaDbPath.Value) || etternaCharts.Count == 0)
                 currentlyCached.RemoveAll(x => x.OriginalGame == OtherGameMapDatabaseGame.Etterna);
 
+            // Make sure there're no duplicate Checksums
+            osuMaps = Helpers.ListHelper.DistinctBy(osuMaps, x => x.Md5Checksum).ToList();
+            etternaCharts = Helpers.ListHelper.DistinctBy(etternaCharts, x => x.Md5Checksum).ToList();
+
+            // Creating hash objects
+            var osuMapsHash = new Dictionary<string, OtherGameMap>();
```

Comment:
Can be done in one line by doing `var osuMapsHash = osuMaps.ToDictionary(x => x.Md5Checksum);`